protocol/x/vault/keeper/orders.go

```diff
@@ -30,10 +32,79 @@ const (
 )
 
 // RefreshAllVaultOrders refreshes all orders for all vaults by
-// TODO(TRA-134)
 // 1. Cancelling all existing orders.
 // 2. Placing new orders.
 func (k Keeper) RefreshAllVaultOrders(ctx sdk.Context) {
```

Comment:
will it be a final PR at the end that ties this to app.go?

Where is this called, would this be in CommitTx or in PrepareProposal?

Reply:
It's already tied (see [here](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/x/vault/abci.go#L12)). At the end of every block, this will be called